qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/ConvFunctionModel.cpp

```diff
@@ -247,14 +272,27 @@ void ConvFunctionModel::removeBackground() {
 bool ConvFunctionModel::hasBackground() const { return m_backgroundType != BackgroundType::None; }
 
 EstimationDataSelector ConvFunctionModel::getEstimationDataSelector() const {
-  return [](const Mantid::MantidVec &, const Mantid::MantidVec &,
-            const std::pair<double, double>) -> DataForParameterEstimation { return DataForParameterEstimation{}; };
+  return [](const Mantid::MantidVec &x, const Mantid::MantidVec &y,
+            const std::pair<double, double> &range) -> DataForParameterEstimation {
+    (void)range;
+
+    auto const maxElement = std::max_element(y.cbegin(), y.cend());
+    if (maxElement == y.cend())
```

Comment:
Will this handle the case where `y` has zero length?

Reply:
Yes this should be handled ok